### PR TITLE
chore: remove devdays banner

### PR DESCRIPTION
**Pull request by florlafuente**
Remove devdays banner

### CODE REVIEW

**Comment by UdoBretz** [comment]
@Sambego Can this be merged, just noticed on jwt.io the banner is still there.

**Review by Sambego — APPROVED** [review]

**Comment by Sambego** [comment]
@UdoBretz apparently this PR wasn't complete yet, so I created a [second one](https://github.com/jsonwebtoken/jsonwebtoken.github.io/pull/659) that removes the banner. As soon as it get's an approval I'll merge and deploy it myself.

**Pull request merged**
